lib/tasks/maintenance.rb

```diff
@@ -0,0 +1,11 @@
+namespace :maintenance do
+  desc "Start maintenance mode"
+  task start: :environment do
```

Comment:
Oh, this is just the db:migrate issue, let me merge the latest master to fix it.